tests/mdev_configuration_allocation_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"time"
 
 	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/libnet"
```

Comment:
to move those as well to last block ? 
maybe even the first one, so it stop adding it there (visual studio code auto added it here)